### PR TITLE
P16 and P16a glider recoveries

### PR DESCRIPTION
**Pull request by cdobs**
Updating recovery information for gliders recovered on P16 and P16a

### CODE REVIEW

File: deployment/CP05MOAS-GL559_Deploy.csv [hunk 1]
```diff
@@ -1,7 +1,7 @@
...
+TS20210131,,TS20210419,,CP05MOAS-GL559-00-ENG000000,1,1,2021-01-31T18:01:00,2021-04-19T17:44:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00559,CGCON-GLDRCP-00559,39.92493,-71.1781,,1000,1000,FZ-1,,
+TS20210131,,TS20210419,,CP05MOAS-GL559-01-ADCPAM000,1,1,2021-01-31T18:01:00,2021-04-19T17:44:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00559,CGINS-ADCPAM-43106,39.92493,-71.1781,,1000,1000,FZ-1,,
+TS20210131,,TS20210419,,CP05MOAS-GL559-02-FLORTM000,1,1,2021-01-31T18:01:00,2021-04-19T17:44:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00559,CGINS-FLORTM-03921,39.92493,-71.1781,,1000,1000,FZ-1,,
+TS20210131,,TS20210419,,CP05MOAS-GL559-03-CTDGVM000,1,1,2021-01-31T18:01:00,2021-04-19T17:44:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00559,CGINS-CTDGVM-09029,39.92493,-71.1781,,1000,1000,FZ-1,,
+TS20210131,,TS20210419,,CP05MOAS-GL559-04-DOSTAM000,1,1,2021-01-31T18:01:00,2021-04-19T17:44:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00559,CGINS-DOSTAM-00436,39.92493,-71.1781,,1000,1000,FZ-1,,
+TS20210131,,TS20210419,,CP05MOAS-GL559-05-PARADM000,1,1,2021-01-31T18:01:00,2021-04-19T17:44:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00559,CGINS-PARADM-50145,39.92493,-71.1781,,1000,1000,FZ-1,,
```
Comment (rtravis-whoi) [inline deployment/CP05MOAS-GL559_Deploy.csv:7]: looks good; recovery cruise is in cruiseinformation.csv

**Review by rtravis-whoi — APPROVED** [review]

File: deployment/CP05MOAS-GL339_Deploy.csv [hunk 2]
```diff
@@ -1,79 +1,79 @@
...
+AR52,,AR52,,CP05MOAS-GL339-00-ENG000000,13,1,2021-04-02T13:34:00,2021-04-16T15:26:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00339,CGCON-GLDRCP-00339,40.07595,-70.383683,,200,200,EB,,
+AR52,,AR52,,CP05MOAS-GL339-01-ADCPAM000,13,1,2021-04-02T13:34:00,2021-04-16T15:26:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00339,CGINS-ADCPAM-50865,40.07595,-70.383683,,200,200,EB,,
+AR52,,AR52,,CP05MOAS-GL339-02-FLORTM000,13,1,2021-04-02T13:34:00,2021-04-16T15:26:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00339,CGINS-FLORTM-04472,40.07595,-70.383683,,200,200,EB,,
+AR52,,AR52,,CP05MOAS-GL339-03-CTDGVM000,13,1,2021-04-02T13:34:00,2021-04-16T15:26:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00339,CGINS-CTDGVM-09338,40.07595,-70.383683,,200,200,EB,,
+AR52,,AR52,,CP05MOAS-GL339-04-DOSTAM000,13,1,2021-04-02T13:34:00,2021-04-16T15:26:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00339,CGINS-DOSTAM-00161,40.07595,-70.383683,,200,200,EB,,
+AR52,,AR52,,CP05MOAS-GL339-05-PARADM000,13,1,2021-04-02T13:34:00,2021-04-16T15:26:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00339,CGINS-PARADM-50242,40.07595,-70.383683,,200,200,EB,,
```
[the remark this answers is not in the document]
Comment (snwhite) [reply deployment/CP05MOAS-GL339_Deploy.csv:79]: Looks good.

File: deployment/CP05MOAS-GL379_Deploy.csv [hunk 3]
```diff
@@ -1,67 +1,67 @@
...
+TS20210131,,TS20210419,,CP05MOAS-GL379-00-ENG000000,11,1,2021-01-31T17:57:00,2021-04-19T18:14:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGCON-GLDRCP-00379,39.92648,-71.17723,,1000,1000,SS-2,,
+TS20210131,,TS20210419,,CP05MOAS-GL379-01-ADCPAM000,11,1,2021-01-31T17:57:00,2021-04-19T18:14:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-ADCPAM-49697,39.92648,-71.17723,,1000,1000,SS-2,,
+TS20210131,,TS20210419,,CP05MOAS-GL379-02-FLORTM000,11,1,2021-01-31T17:57:00,2021-04-19T18:14:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-FLORTM-05133,39.92648,-71.17723,,1000,1000,SS-2,,
+TS20210131,,TS20210419,,CP05MOAS-GL379-03-CTDGVM000,11,1,2021-01-31T17:57:00,2021-04-19T18:14:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-CTDGVM-09453,39.92648,-71.17723,,1000,1000,SS-2,,
+TS20210131,,TS20210419,,CP05MOAS-GL379-04-DOSTAM000,11,1,2021-01-31T17:57:00,2021-04-19T18:14:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-DOSTAM-00171,39.92648,-71.17723,,1000,1000,SS-2,,
+TS20210131,,TS20210419,,CP05MOAS-GL379-05-PARADM000,11,1,2021-01-31T17:57:00,2021-04-19T18:14:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-PARADM-50277,39.92648,-71.17723,,1000,1000,SS-2,,
```
[the remark this answers is not in the document]
Comment (snwhite) [reply deployment/CP05MOAS-GL379_Deploy.csv:67]: Looks good.  Cruise ID is in CruiseInformation, and date/time matches draft of P16a QLR.

File: deployment/CP05MOAS-GL559_Deploy.csv (hunk 1 above)
Comment (snwhite) [reply deployment/CP05MOAS-GL559_Deploy.csv:7]: Looks good.  Cruise ID is in CruiseInformation, and date/time matches draft of P16a QLR.

**Review by snwhite — APPROVED** [review]

**Pull request merged**